lib/aptible/cli/subcommands/apps.rb

```diff
@@ -94,6 +94,28 @@ def apps
                 raise if e.response.status != 404
               end
             end
+
+            desc 'apps:rename OLD_HANDLE NEW_HANDLE [--environment'\
+                 ' ENVIRONMENT_HANDLE]', 'Rename an app handle. In order'\
+                 ' for the new app handle to appear in log drain and metric'\
+                 ' drain destinations, you must restart the app.'
+            option :environment
+            define_method 'apps:rename' do |old_handle, new_handle|
+              env = ensure_environment(options)
+              app = ensure_app(options.merge(app: old_handle))
+              app.update(handle: new_handle)
+              m1 = "In order for the new app name (#{new_handle}) to appear"\
+                   ' in log drain and metric drain destinations, you must'\
+                   ' restart the app.'
+              m2 = 'You can restart your app with this command: "aptible'\
+                   " restart --app #{new_handle} --environment #{env.handle}\""
+              m3 = 'Warning - Git remote addresses must be updated to match'\
+                   ' the new handle, if using Dockerfile deploy. '\
+                   "(git@beta.aptible.com:madhu-sandbox/#{new_handle}.git)"
```

Comment:
`madhu-sandbox` seems to be a placeholder, it should be something like `#{app.account.handle}`?

Reply:
oof, good find